### PR TITLE
chore: update root package.json configuration

### PR DESCRIPTION
**Pull request by flaviosalesv**
Updated the package.json to include necessary changes for project dependencies and configuration. Please review and let me know if any further adjustments are needed.

## Summary by CodeRabbit

- **New Features**
	- Introduced a new script for managing ngrok configuration for local development.
	- Added a new dependency for improved environment variable management.

- **Improvements**
	- Enhanced local development setup for better handling of environment variables with the addition of `dotenv-cli`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,14 +17,17 @@
...
+    "test": "vitest run",
+    "_ngrok": "ngrok http 8000 --domain $NGROK_DOMAIN",
+    "ngrok": "dotenv -e ./apps/api/.env -- npm run _ngrok"
   },
   "dependencies": {
     "devcert": "^1.2.2",
     "rimraf": "^5.0.5"
   },
   "devDependencies": {
+    "cross-env": "^7.0.3",
```
Comment (sweetrdev) [inline package.json:29]: Do we need this dependency? I don't see it being used.

**Review by sweetrdev — COMMENTED** [review]
Nice job :) leaving one question.

File: package.json [hunk 2]
```diff
@@ -17,10 +17,12 @@
...
     "heroku-postbuild": "turbo run build --filter api",
     "start": "cd apps/api && npm run start",
-    "ngrok": "ngrok http https://api.sweetr.local:8000 --domain weekly-flying-primate.ngrok-free.app",
-    "test": "vitest run"
+    "test": "vitest run",
+    "_ngrok": "ngrok http 8000 --domain $NGROK_DOMAIN",
+    "ngrok": "dotenv -e ./apps/api/.env -- npm run _ngrok"
   },
   "dependencies": {
+    "@tabler/icons-react": "^3.19.0",
```
Comment (sweetrdev) [inline package.json:25]: A couple things:
- I think you added tabler icons here by mistake?
- Didn't we need `dotenv-cli`? Isn't that package the one providing the `dotenv` binary used by the `ngrok` command?

**Review by sweetrdev — COMMENTED** [review]
Thanks for the update!

Seems there a couple more things to adjust. Also leaving a question.

**Review by waltergalvao — APPROVED** [review]
Nice!

**Pull request merged**